go/client/cmd_kbfs_mount.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+// +build !darwin
```

Comment:
@zanderz last thing, should this be `// +build windows` instead?  I.e., do we want this to build on Linux?